From 0dd619a35c97a8886df0e0c9e06e5df7c637dd2c Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@toradex.com>
Date: Thu, 28 Apr 2022 06:21:08 -0300
Subject: [PATCH 1/3] update default grub cfg header

Original title: "ostree: cherry-pick patches from meta-lmp-base"

Original description:

  This commit is part of the work to remove the dependency on
  meta-lmp-base.

  Related-to: TOR-2247

Upstream-Status: Inappropriate [embedded specific]

  Signed-off-by: Sergio Prado <sergio.prado@toradex.com>
---
 src/boot/grub2/ostree-grub-generator | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/boot/grub2/ostree-grub-generator b/src/boot/grub2/ostree-grub-generator
index d1436b65..3caebda6 100644
--- a/src/boot/grub2/ostree-grub-generator
+++ b/src/boot/grub2/ostree-grub-generator
@@ -99,8 +99,8 @@ populate_header()
 {
 cat >> ${new_grub2_cfg} <<EOF
 serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
-default=boot
-timeout=10
+default=0
+timeout=1
 
 EOF
 }
-- 
2.25.1

